src/apps/backend/app.ts

```diff
@@ -39,9 +52,43 @@ export default class App {
       Logger.info('app - server started listening on  - %s', server.address());
     });
 
+    DocumentationService.generateAndInjectDocumentationUnderDistFolder().catch(
```

Comment:
rename `generateAndInjectDocumentationUnderDistFolder` to `generateAPIDocumentation`. Dist folder is somewhat an internal concern of the service.